pkg/oc/admin/diagnostics/diagnostics/client/config_contexts.go

```diff
@@ -154,6 +154,9 @@ var (
 
 // Name is part of the Diagnostic interface and just returns name.
 func (d ConfigContext) Name() string {
+	if d.ContextName == "" {
```

Comment:
nit: len(d.ContextName) == 0

Reply:
I looked up a long gonuts thread on which way was better, and the consensus was essentially "whichever looks clearest to you". I can look that up if you want :) Do we have a coding standard on this that says otherwise?

Reply:
Needless to say, I think comparing to empty string is less to parse than checking length against 0.

Reply:
> Do we have a coding standard on this that says otherwise?

Not really a coding standard so much as a cli convention (most likely with a few cases here and there that have been missed :) )